dashboard/src/components/AppView/AppControls/RollbackButton/RollbackDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import LoadingWrapper from "../../../LoadingWrapper";
+
+import "./RollbackDialog.css";
+
+interface IRollbackDialogProps {
+  loading: boolean;
+  revision: number;
```

Comment:
You have both `props.revision` and `state.revision` which I found confusing. Perhaps the prop could be `currentRevision` and the state `targetRevision`? (would have avoided confusion for me, at least).